BSB_lan.ino

```diff
@@ -9028,6 +9078,9 @@ void setup() {
 
   pinMode(LED_BUILTIN, OUTPUT);
 
+  #ifdef ESP32
+  EEPROM.begin(2048); // size in Byte
```

Comment:
EEPROM size on the 2560 as well as the EEPROMs used on the board are 4096 byte of size. Does the ESP32 not provide that much space?

Reply:
ESP32 doesn't provide EEPROM, instead the Flash can be used as a EEPROM emulation layer. So this value can be changed

Reply:
Ok, then 4096 it should be.